src/main/java/vazkii/psi/common/item/ItemCAD.java

```diff
@@ -202,7 +197,7 @@ public ActionResultType onItemUse(ItemUseContext ctx) {
 				setCADComponent(playerCad, dyeStack);
 			}
 		}
-		boolean did = cast(worldIn, playerIn, data, bullet, itemStackIn, 40, 25, 0.5F, ctx -> ctx.castFrom = hand);
+		boolean did = null != cast(worldIn, playerIn, data, bullet, itemStackIn, 40, 25, 0.5F, ctx -> ctx.castFrom = hand);
```

Comment:
nitpick but change the order of the condition

Reply:
Already resolved, comment made in reference of outdated commit.